crates/rpc-types-debug/src/lib.rs

```diff
@@ -5,6 +5,9 @@
 )]
 #![cfg_attr(not(test), warn(unused_crate_dependencies))]
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
+#![no_std]
```

Comment:
not sure if we can make this entirely no-std

Reply:
Oh can you say why? I think right now it can be, but maybe you mean in the future?

Reply:
oh actually we only have one type in here, so this seems fine